docs/tre-workspace-authors/firewall-rules.md

```diff
@@ -1,128 +1,256 @@
-# Adding Firewall Rules as part of a workspace or service deployment
+# Configuring Firewall Rules in Azure TRE
 
-!!! note
-    Creating firewall rules will in the future only be allowed through the firewall shared services [#882](https://github.com/microsoft/AzureTRE/issues/882).
+This guide explains how to configure firewall rules for your Azure TRE environment.
```

Comment:
This is specific to creating a workspace or service, as per the original document, nto for TRE as a whole.